core/src/main/java/com/scalar/db/storage/dynamo/Namespace.java

```diff
@@ -0,0 +1,38 @@
+package com.scalar.db.storage.dynamo;
+
+public class Namespace {
```

Comment:
I added this object to help manipulate "namespace" that can be prefixed or nonprefixed in the DynamoAdmin.